packages/carbon-web-components/package.json

```diff
@@ -192,7 +192,7 @@
     "rollup-plugin-multi-input": "^1.3.1",
     "rollup-plugin-terser": "^7.0.2",
     "rtlcss": "^3.0.0",
-    "sass": "~1.57.0",
+    "sass": "~1.40.0",
```

Comment:
@ariellalgilmore did you mean to bump `sass` down?

Reply:
@kennylam yeah `sass` has an error with `@extend` causing the styles to break. but actually they just released 1.58 which  should hopefully fix it. https://github.com/sass/dart-sass/releases

`Fix a bug in @extend logic where certain selectors with three or more combinators were incorrectly considered superselectors of similar selectors with fewer combinators, causing them to be incorrectly trimmed from the output.`

Reply:
Gotcha. Another thing we could try is just to keep in sync with the v11 sass version (`1.51.0`). But as long as it's working now then all good.